Assets/Stack mini game/Scripts/GameplayManager.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GameplayManager : MonoBehaviour
```

Comment:
I think we should prefix these class names with StackMinigame or put them in a namespace as they are only for this minigame not the wider code

Reply:
Done